patches/api/0382-Add-PlayerReleaseItemEvent.patch

```diff
@@ -0,0 +1,64 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: u9g <git@u9g.dev>
+Date: Tue, 3 May 2022 20:41:30 -0400
+Subject: [PATCH] Add PlayerReleaseItemEvent
+
+
+diff --git a/src/main/java/com/destroystokyo/paper/event/player/PlayerReleaseItemEvent.java b/src/main/java/com/destroystokyo/paper/event/player/PlayerReleaseItemEvent.java
+new file mode 100644
+index 0000000000000000000000000000000000000000..b3b85d54090c567ae05bf68809c47c24ea036e8a
+--- /dev/null
++++ b/src/main/java/com/destroystokyo/paper/event/player/PlayerReleaseItemEvent.java
+@@ -0,0 +1,52 @@
++package com.destroystokyo.paper.event.player;
++
++import org.bukkit.entity.Player;
++import org.bukkit.event.HandlerList;
++import org.bukkit.event.player.PlayerEvent;
++import org.bukkit.inventory.ItemStack;
++import org.jetbrains.annotations.NotNull;
++
++/**
++ * Called when the server detects a player releasing an item.
++ */
++public class PlayerReleaseItemEvent extends PlayerEvent {
++    private static final HandlerList handlers = new HandlerList();
++    @NotNull private final ItemStack item;
++    private final int useItemRemainingTicks;
++
++    public PlayerReleaseItemEvent(@NotNull final Player player, @NotNull final ItemStack item, final int useItemRemainingTicks) {
++        super(player);
++        this.item = item;
++        this.useItemRemainingTicks = useItemRemainingTicks;
++    }
++
++    /**
++     * Gets the item the player is releasing
++     *
++     * @return ItemStack the item the player released
++     */
++    @NotNull
++    public ItemStack getItem() {
++        return item;
++    }
++
++    /**
++     * Gets the number of ticks the item was held for
++     *
++     * @return int the number of ticks the item was held for
++     */
++    public int getUseItemRemainingTicks() {
```

Comment:
"item remaing" and the "tickets the item was held for" doesn't really line up

Reply:
renamed to `getTicksHeldFor()`